connectors/spark/src/main/scala/io/unitycatalog/spark/UCSingleCatalog.scala

```diff
@@ -48,7 +49,24 @@ class UCSingleCatalog extends TableCatalog with SupportsNamespaces {
       columns: Array[Column],
       partitions: Array[Transform],
       properties: util.Map[String, String]): Table = {
-    deltaCatalog.createTable(ident, columns, partitions, properties)
+    val hasExternalClause = properties.containsKey(TableCatalog.PROP_EXTERNAL)
+    val hasLocationClause = properties.containsKey(TableCatalog.PROP_LOCATION)
+    def isPathTable = ident.namespace().length == 1 && new Path(ident.name()).isAbsolute
```

Comment:
In what scenario is `ident` a path? Is it when user says:

`df.format("delta").save("<path>")` ?

Reply:
it's something like `CREATE TABLE delta.'/my/path' ...`